README.md

```diff
@@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
 
 Only simple products are synced. Configurable products are not synced.
 
+## Requirements
+
+In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
+declared in the crontab via the following:
+
+php cron.php -mdefault 1
```

Comment:
Ah, of course, I had taken for granted that the schedule syntax would be *
* * * , was focused more on the command aspect. I'll be sure to include
that in the README.md update

On Fri, Aug 21, 2015 at 2:18 PM, Yan Pritzker <notifications@github.com>
wrote:

> In README.md
> <https://github.com/reverbdotcom/magento/pull/3#discussion_r37668671>:
>
> > @@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
> >
> >  Only simple products are synced. Configurable products are not synced.
> >
> > +## Requirements
> > +
> > +In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
> > +declared in the crontab via the following:
> > +
> > +php cron.php -mdefault 1
>
> But...this is not cron syntax? Am I missing something? Shouldn't every
> line start with timing information? "* * * * *"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reverbdotcom/magento/pull/3/files#r37668671>.
>